policy/modules/contrib/virt.te

```diff
@@ -1355,6 +1355,10 @@ optional_policy(`
 	xserver_rw_shm(virt_domain)
 ')
 
+optional_policy(`
```

Comment:
if it uses types from the same module, it does not need to be in an optional block. Actually using directly stream_connect_pattern() would be better.

Reply:
This is the 'virt' module using types from the 'qgs' module, so surely 'optional_policy' is needed here ?

Reply:
You are right, I overlooked this actually is in virt.te.